libs/ui-watt/src/lib/components/slider/watt-slider.component.ts

```diff
@@ -52,33 +56,86 @@ export class WattSliderComponent {
   /** Step between each value. */
   @Input() step = 1;
 
-  /** Manually define all steps. Overrules `min`, `max` and `step`. */
-  @Input() customSteps?: number[];
-
   /** The currently selected range value. */
   @Input() value: WattSliderValue = { min: this.min, max: this.max };
 
+  @ViewChild('maxRange') maxRange!: ElementRef<HTMLInputElement>;
+
+  @ViewChild('minRange') minRange!: ElementRef<HTMLInputElement>;
+
+  private _maxChangeSubscription!: Subscription;
+  private _minChangeSubscription!: Subscription;
+  private _providedStep = this.step;
```

Comment:
issue: the `_providedStep` will be set to 1 initially (default value of `step`) and will not be updated when new values arrive in `step`. In order to update `_providedStep `, `onChanges` hook can be used (or the `step` Input is converted to a setter).